src/components/data-exploration/differential-expression-tool/LaunchPathwayAnalysisModal.jsx

```diff
@@ -137,13 +139,18 @@ const LaunchPathwayAnalysisModal = (props) => {
       </Modal>
       {
         advancedFilteringOpen && (
-          <AdvancedFilteringModal onCancel={() => setAdvancedFilteringOpen(false)} />
+          <AdvancedFilteringModal 
+          onLaunch={onApplyFilters} 
+          onCancel={() => setAdvancedFilteringOpen(false)} 
```

Comment:
We should try to have consistent names on the modal handler `useState`'s.

Up to now I think we have used `...ModalVisible`, so I would say use that for `AdvancedFilteringModal` and for `LaunchPathwayAnalysisModal`